content/the-ember-times-issue-179.md

```diff
@@ -26,15 +28,25 @@ tags:
 
 ---
 
-## [2. Section title in sentence case 🐹](section-url)
+## Readers' Question: What’s the current status of the Embroider project? 🧵
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+<Amy's note: Am wondering if this would be better to publish when [https://github.com/thoov/rfcs/blob/enable-embroider/text/0000-enable-embroider.md](https://github.com/thoov/rfcs/blob/enable-embroider/text/0000-enable-embroider.md) is live on the emberjs/rfcs repo?>
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+What's the current status of the Embroider project? [Embroider](https://github.com/embroider-build/embroider) is nearing a 1.0 release. The official cutover point will be when Ember CLI starts generating new apps with Embroider by default. 🙌
+
+If you're not familiar with Embroider, Embroider is a new build pipeline that will provide a way to compile Ember apps into spec-compliant, modern JavaScript. Using Embroider will make it easier to take advantage of code splitting and tree shaking in your app.
+
+Several large, heavily-tested Ember apps are already shipping to production with Embroider! However, there are some risks if you choose to use Embroider in production today. (You're likely to discover that some Ember addons don't work or will break your build. Also, Embroider's own configuration options are subject to change, so you'll need to read the [CHANGELOG.md](https://github.com/embroider-build/embroider/blob/master/CHANGELOG.md) to check for any gotchas when updating Embroider packages in your app.)
```

Comment:
When some addons in one's app aren't compatible with Embroider, is there a tip that we can provide to readers? (I ask because I don't know the answer and would like to know if I were in a reader's shoes.)

Reply:
AFAIK it's a showstopper, you can't use Embroider then. I think CSS Modules is one of the more common incompatible addons.